tg.js

```diff
@@ -507,20 +507,20 @@ windows.controls.drop Patak
 windows.controls.wepVis I-toggle ang Visibility ng Armas
 windows.controls.kickVoteY Tanggalin mula sa laro [Oo]
 windows.controls.kickVoteN Tanggalin mula sa laro [Hindi]
-windows.controls.kpdVoteY Vote sa KPD [Oo]
-windows.controls.kpdVoteN Vote sa KPD [Hindi]
+windows.controls.kpdVoteY Botohin sa KPD [Oo]
```

Comment:
For lines 510 & 511 could it be "bumoto"?

Reply:
Absolutely. Thanks.